crates/re_viewer/src/app.rs

```diff
@@ -329,7 +329,7 @@ impl App {
                         self.add_receiver(rx);
                     }
                     Err(err) => {
-                        re_log::error!("Failed to open data source: {err}");
+                        re_log::error!("Failed to open data source: {}", re_error::format(err));
```

Comment:
This is a really bad gotcha. By default, `anyhow::Error` only shows the latest part of its context 🤦